.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
 node_js:
   - "0.10"
+  - "0.12"
 script: make travis
```

Comment:
Bang and perms, but it would still need to be './make'.

Sent from my iPad

> On Mar 5, 2015, at 2:37 AM, XhmikosR <notifications@github.com> wrote:
> 
> In .travis.yml:
> 
> > @@ -1,4 +1,7 @@
> >  language: node_js
> >  node_js:
> >    - "0.10"
> > +  - "0.12"
> >  script: make travis
> In theory doing make should work on nix if the proper shebang is present, shouldn't it?
> …
> —
> Reply to this email directly or view it on GitHub.
>